pkg/crc/machine/start.go

```diff
@@ -171,22 +174,13 @@ func runGrowpart(sshRunner *crcssh.Runner, rootPart string) error {
 	return nil
 }
 
-func growLVForMicroshift(sshRunner *crcssh.Runner, lvFullName string) error {
-	rootPart, _, err := sshRunner.RunPrivileged("Get PV disk path", "/usr/sbin/pvs", "-S", fmt.Sprintf("lv_full_name=%s", lvFullName), "-o", "pv_name", "--noheadings")
-	if err != nil {
-		return err
-	}
-	rootPart = strings.TrimSpace(rootPart)
-	if err := runGrowpart(sshRunner, rootPart); err != nil {
-		return err
-	}
-
-	if _, _, err := sshRunner.RunPrivileged("Resizing the physical volume(PV)", "/usr/sbin/pvresize", rootPart); err != nil {
+func growLVForMicroshift(sshRunner *crcssh.Runner, lvFullName string, rootPart string) error {
+	if _, _, err := sshRunner.RunPrivileged("Resizing the physical volume(PV)", "/usr/sbin/pvresize", "--devices", rootPart, rootPart); err != nil {
 		return err
 	}
 
 	// Get the size of volume group
-	sizeVG, _, err := sshRunner.RunPrivileged("Get the volume group size", "/usr/sbin/vgs", "--noheadings", "--nosuffix", "--units", "b", "-o", "vg_size")
+	sizeVG, _, err := sshRunner.RunPrivileged("Get the volume group size", "/usr/sbin/vgs", "--noheadings", "--nosuffix", "--units", "b", "-o", "vg_size", "--devices", rootPart)
```

Comment:
"/sysroot" ?

Everywhere else in the PR this has been removed/changed. I am not sure why that is... as it seems still relevant.